app/ui.js

```diff
@@ -1644,7 +1644,7 @@ const UI = {
 };
 
 // Set up translations
-const LINGUAS = ["cs", "de", "el", "es", "ko", "nl", "pl", "sv", "tr", "zh_CN", "zh_TW"];
+const LINGUAS = ["ru_RU", "cs", "de", "el", "es", "ko", "nl", "pl", "sv", "tr", "zh_CN", "zh_TW"];
```

Comment:
Aside from changing this to 'ru' could you please also place it alphabetically in this list?

Reply:
Strange, I have everything on the contrary, tested in Firefox and Chrome, if "ru", then does not work.

Reply:
If you open a console in your browser and type navigator.language, what do you get?

Reply:
![image](https://user-images.githubusercontent.com/14820301/53351046-ee925f80-3939-11e9-8414-25952a651792.png)

Reply:
Firefox in Linux:
![image](https://user-images.githubusercontent.com/14820301/53352002-7a58bb80-393b-11e9-8323-6ffa8ab4ce07.png)

Reply:
![image](https://user-images.githubusercontent.com/14820301/53352759-d4a64c00-393c-11e9-96db-ed2a82daea5d.png)

Reply:
Ok. So that explains why the translations are working for you. Your `navigator.language` string is different from mine. But it doesn't explain why only `ru` in the code wouldn't work.

The issue here is that if we run your current code, people running Linux will not be getting russian translations in our browsers, since our `navigator.language` value is simply `ru`. Another issue is that anyone whose browser is configured to use a sub language of russian (for example Moldavian russian, `ru-md`) won't get the fallback to normal russian translations if our code is specified for `ru_RU`.

Are you sure that, when you have tried `ru`, you have made all the necessary changes?

 * rename the po-file to `ru.po`
 * change `ru_RU` to `ru` in `po/Makefile` and `app/ui.js`
 * go to `po/` and run `make update-pot` and then `make update-js`

It should work.

Reply:
Make and PR